source/MaterialXGenShader/Util.cpp

```diff
@@ -75,11 +75,11 @@ void getSubDirectories(const std::string& baseDirectory, StringVec& relativePath
 #endif
 }
 
-void getDocumentsInDirectory(const std::string& directory, StringVec& files)
+void getFilesInDirectory(const std::string& directory, StringVec& files, const std::string& extension)
```

Comment:
Change to allow scanning for files with any extension. i.e. make routine more generic.